### DISCUSSION TITLE
Activating Debug Mode

### DISCUSSION THREAD

**Discussion by ericktreichel** [johannschopplich/nuxt-gtag#69] [General]
Hi,
Is there a way to activate debug_mode using nuxt-gtag?
Per Analytics docs (https://developers.google.com/analytics/devguides/collection/ga4/set-up-ecommerce#verify-data) we would do this way:
`gtag('config', 'TAG_ID',{ 'debug_mode': true });`
where debug_mode: true is passed in the option object for additional configs.
Can we use it now? If note, do you thing would be a valid feature to implement in the module?
Thanks in advance,
Erick T

**Comment by johannschopplich** [comment]
You can do so by adding the config command to the module options:

```ts
export default defineNuxtConfig({
  modules: ['nuxt-gtag'],

  gtag: {
    id: 'G-XXXXXXXXXX',
    // Additional configuration for this tag ID
    config: {
      debug_mode: true
    },
})
```
